src/data/imprint/index.tsx

```diff
@@ -13,6 +13,6 @@ export interface Imprint {
 const imprint: TranslationCollection<Imprint> = data
 
 export const getImprint = (locale: Locale): Imprint =>
-  getTranslation(locale)(imprint)
+  filterByLocale(locale)(imprint)
```

Comment:
I'd leave this at
```suggestion
  extractSingleTranslation(locale)(imprint)
```
for now, to preserve the type checking

Reply:
Done